tests/helper.js

```diff
@@ -8,10 +11,9 @@
   exports.apiProduct = {guid: 'server-guid', name: "Name from API",
                         logo_url: "img.jpg"};
 
-
   exports.setUp = function setUp() {
     exports.server = sinon.fakeServer.create();
-    exports.settings = fxpay.settings.configure({
+    exports.settings = settings.configure({
```

Comment:
Hmm, yeah I removed the things that reference that. So it should be ok to remove it.